src/pane-element.coffee

```diff
@@ -156,5 +156,7 @@ atom.commands.add 'atom-pane',
   'pane:split-down': -> @getModel().splitDown(copyActiveItem: true)
   'pane:close': -> @getModel().close()
   'pane:close-other-items': -> @getModel().destroyInactiveItems()
+  'pane:enlarge': -> @getModel().setFlexScale(@getModel().getFlexScale() * 1.1)
+  'pane:shrink': -> @getModel().setFlexScale(@getModel().getFlexScale() / 1.1)
```

Comment:
What do you think about renaming these to `pane:increase-size` and `pane:decrease-size` to be more consistent with the command names that adjust the font size?